src/main/java/core/basesyntax/service/impl/FileOperationImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.FruitRecordDtoParser;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.StorageService;
+import java.util.List;
+import java.util.Map;
+
+public class FileOperationImpl implements OperationStrategy {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String INPUT_FILE_PATH = "src/main/resources/fruitStorageInfo.csv";
```

Comment:
pass input file path to method from main

Reply:
then I need to delete method doOperation? Because I will realize all logic in main?

Reply:
up to you